src/features/folder_creation/manual_contest_folder.ts

```diff
@@ -34,9 +34,11 @@ export const manualContestFolderCreation = async (
     }
     catch(err) {
         if(err.code === ErrorCodes.folderExists) {
-            vscode.window.showErrorMessage('Contest folder already exists');
-        }else if(err.code ===ErrorCodes.noWritePermission) {
+            vscode.window.showErrorMessage("Contest folder already exists");
+        }else if(err.code ===ErrorCodes.noAccessPermission) {
             vscode.window.showErrorMessage("Please open a folder in your workspace");
```

Comment:
Error message here looks good, would be good to have same error message ("Please open a folder in your workspace") in problem folder creations as well